.prettierrc

```diff
@@ -2,6 +2,7 @@
   "useTabs": false,
   "singleQuote": false,
   "trailingComma": "es5",
+  "endOfLine": "auto",
```

Comment:
Let's use `lf` please, which should evaluate to the same result but is more explicit.

Reply:
`lf` is the default for `endOfLine`, this exactly what creates the problem for Windows (I already tried to use `lf`, but it doesn't work)